src/signals/incident/components/form/DescriptionWithClassificationInput/index.test.js

```diff
@@ -122,5 +123,25 @@ describe('Form component <DescriptionWithClassificationInput />', () => {
         'input-field-name': '',
       });
     });
+
+    it('doesn\'t call the predictions when they are disabled', () => {
+      wrapper.setProps({
+        meta: {
+          ...metaFields,
+          isVisible: true,
+        },
+        parent: {
+          ...parent,
+          meta: {
+            ...parent.meta,
+            incidentContainer: { usePredictions: false },
+          },
+        },
+      });
+
+      wrapper.find(TextArea).simulate('blur', { target: { value: 'diabolo' } });
+      expect(parent.meta.getClassification).not.toHaveBeenCalled();
+      expect(parent.meta.updateIncident).not.toHaveBeenCalledWith();
```

Comment:
I expect you mean to use `toHaveBeenCalled` here as well, like the line above?

Reply:
Indeed.